### ISSUE TITLE
UntilInsteadOfRangeTo doesn't report when `rangeTo` is used

### ISSUE THREAD

**Issue by 3flex** [detekt/detekt#4207]
## Expected Behavior of the rule
Both of these uses of `rangeTo` should be flagged:
```kotlin
        for (i in 0.rangeTo(10 - 1)) {}
        val range = 0.rangeTo(10 - 1)
```

## Context
The example above is semantically equivalent to:
```kotlin
        for (i in 0..10 - 1) {}
        val range = 0..10 - 1
```

Which is flagged as expected.
